configure.ac

```diff
@@ -16,7 +19,7 @@ AC_TYPE_SSIZE_T
 
 # Checks for library functions.
 AC_FUNC_MALLOC
-AC_CHECK_FUNCS([socket strdup strerror])
+AC_CHECK_FUNCS([socket strdup strerror vasprintf asprintf], [], [exit 1])
```

Comment:
I'm not fully sure `exit 1` is the right way to make it abort on error. I'm open to suggestions.